tests/acceptance/features/apiSpaces/shareSubItemOfSpaceViaPublicLink.feature

```diff
@@ -117,3 +117,17 @@ Feature: Share a file or folder that is inside a space via public link
       | ocs_api_version | ocs_status_code |
       | 1               | 100             |
       | 2               | 200             |
+
+
+  Scenario Outline: An user participant of the project space can see the created public resources link
+    Given user "Alice" has shared a space "share sub-item" to user "Brian" with role "<spaceRole>"
+    And user "Alice" has created a public link share inside of space "share sub-item" with settings:
```

Comment:
A `When` step is available for this. so instead of a given we can use when and also ADD https status code check step

Reply:
The scenario is not to create a link share but to check that space members can see the created share so that Victor might have written this scenario this way.

Any thoughts?
CC @individual-it

Reply:
let's change it to a nice Given, When, Then flow